SETUP.md

```diff
@@ -269,3 +269,29 @@ Additionally, you must install the [spark-cosmosdb connector](https://docs.datab
    7. Restart the cluster.
 
 </details>
+
+## Setup guide for Docker
+
+Docker images of the repo for different environment are provided. Dockerfiles can be found at `Recommenders/reco_utils/docker` if one would like to build Docker images on his/her own.
+
+Assuming Docker is pre-installed and configured (**NOTE** `docker` command is already available in the Azure Data Science Virtual Machine. For installation of Docker in on a Linux machine, check instructions [here](https://docs.docker.com/install/)), for example, a PySpark Docker image can be built by the following command line
+>   ```{shell}
+>   cd Recommenders/reco_utils/docker/pyspark
```

Comment:
I've been looking at where other libs put the docker files, both [TF](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/tools/ci_build) and [Pytorch](https://github.com/pytorch/pytorch/tree/master/tools/docker) put it under the folder tools. 

Maybe we should consider renaming scripts to tools, this was discussed several times @bethz

Reply:
+1 on renaming

Reply:
this could just be 
```docker build -t <image_name>/<tag> -f reco_utils/docker/pyspark```

Reply:
+1 on the name `tools`

Reply:
@yueguoguo  is the plan to finally create the `tools` folder and add the docker there?

Reply:
@gramhagen what is the final decision for the location of docker?

Reply:
What do people think about renaming scripts -> tools
then the structure for this would be

```
- tools/
    - docker/
        - Dockerfile
    - repo_metrics/
    - databricks_install.py
    - generate_conda_file.py
```

Reply:
I think this looks good. Alternatively, we may want to put `docker` at the top-level? I see many other repos do this.

Reply:
fair, I've seen this quite a bit too. ok. I'm leaning towards keeping this in docker folder at the top level, that's not too cluttered.

we can rename scripts in a separate PR.